lib/api/filter.rb

```diff
@@ -81,10 +82,13 @@ def parse_filter(filter)
       str_method = is_regex ? methods[:regex] : methods[:default]
 
       filter_value, method = case filter_value
-                             when /^'.*'$/
-                               [filter_value.gsub(/^'|'$/, ''), str_method]
-                             when /^".*"$/
-                               [filter_value.gsub(/^"|"$/, ''), str_method]
+                             when /^['"](.*)['"]$/
```

Comment:
Hi Joe, thanks for the previous update.

This above was bugging me and I didn't know why.  Problem with ['"] ... ['"]   will match '\<stuff\>" which we don't want, can you update the when statement to:

```
when /^'(.*)'$/,  /^"(.*)"$/
```

Thank you.

Reply:
@abellotti Great catch. Will do. Thank you!